README.md

```diff
@@ -1116,10 +1122,60 @@ Limited cross table searches are supported.  If you desire a join across tables
           "targetTable": "table1"
         }
       ]
+    },
+    {
+      "tableName": "table2",
+      "index": [
+        {
+          "fieldName": "anotherfieldYouWantToSearch",
+          "tOps": "ADD",
+          "caseSensitive": false,
+          "removeAccents": false
+        }
+      ],
+      "foreignKeys": [
+        {
+          "fieldName": "table2Id",
+          "targetTable": "table2"
+        }
+      ]
     }
   ]
 }
 ```
+## Cross Table query ambiguity
+There will be some cases where two foreign keys refer to the same table.  This ambiguity has the possibility of leading the join logic down a dead end if the foreign key does not lead to the intended table.  To resolve this there are new properties:
```

Comment:
The second and third sentence should be reduced to:

    This ambiguity is resolved by these properties: